packages/frontend/src/build/config/common.ts

```diff
@@ -26,7 +26,7 @@ export const common: Omit<Config, 'backend'> = {
     tvlBreakdown: true,
     diffHistory: false,
     buildAllProjectPages: false,
-    governancePage: false,
+    governancePage: true,
```

Comment:
I think we should still have it disabled when merging.

Reply:
yup, I remember about this - had to enable it since staging was failing with TVL